common/src/main/java/com/wynntils/overlays/StatusEffectsOverlay.java

```diff
@@ -4,6 +4,8 @@
  */
 package com.wynntils.overlays;
 
+import static java.lang.Math.max;
```

Comment:
Oh, sorry I did not notice this before. We don't allow using static imports.

Reply:
Ah I'll fix that quickly. I should get my IDE configured to follow some of these rules